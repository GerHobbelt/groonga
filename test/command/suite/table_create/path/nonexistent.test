table_create Data TABLE_NO_KEY --path nonexistent/directory/data
