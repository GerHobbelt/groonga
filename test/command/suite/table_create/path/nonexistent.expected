table_create Data TABLE_NO_KEY --path nonexistent/directory/data
[
  [
    [
      -3,
      0.0,
      0.0
    ],
    "system call error: No such file or directory: failed to open file info path: <nonexistent/directory/data>"
  ],
  false
]
#|e| system call error: No such file or directory: failed to open file info path: <nonexistent/directory/data>
